packages/truffle-contract/test/sources/Example.sol

```diff
@@ -88,6 +89,10 @@ contract Example {
     assert(false);
   }
 
+  function triggerRequireWithReasonError(){
+    require(false, 'reasonstring');
+  }
```

Comment:
Can we also test a reason string that exceeds 32 bytes? Pardon my having peeked behind the Solidity curtain... ([gist](https://gist.github.com/gnidan/b1890c68c8e0825d4a699929ccb2018e#string-smaller-than-wordsize), if you recall)

Reply:
Good idea! Added a test for this - it turns out that it 'just works' but . . . this is all ganache and more or less improvised. We'll need to revisit the implementation when the clients agree how to *actually* pass this info back.